main.js

```diff
@@ -24,3 +22,144 @@ form.addEventListener('submit', (event) => {
     form.action = 'https://formspree.io/f/moqrpbja';
   }
 });
+
+const projects = [
+
+  {
+    title: 'Tonic',
+    roles: { company: 'Canopy', position: ['Back End Dev', '2015'] },
+    image: '/image/Snapshoot-Portfolio.png',
+    alt: 'Canopy project screenshot',
+    description: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    skills: ['html', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'http://felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'http://github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['grid-item', 'card'], cardImage: ['card-image'], cardDescription: ['card-description'] },
+    cssStyle: '',
+  },
+
+  {
+    title: 'Multi-Post Stories',
+    roles: { company: 'Facebook', position: ['Full Stack Dev', '2015'] },
+    image: '/image/Snapshoot-Portfolio(3).png',
+    alt: 'Multi-Post stories project screenshot',
+    description: 'Experimental content creation feature that allows users to add to an existing story over the course of a day without spamming their friends.',
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'https://felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'https://github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['grid-item', 'card', 'hide-ruby'], cardImage: ['card-image', 'flex-item-2'], cardDescription: ['card-description', 'flex-item-1'] },
+    cssStyle: 'card-l',
+  },
+
+  {
+    title: 'Facebook 360',
+    roles: { company: 'Facebook', position: ['Full Stack Dev', '2015'] },
+    image: '/image/Snapshoot-Portfolio(4).png',
+    alt: 'Facebook 360 project screenshot',
+    description: 'Exploring the future of media in Facebook\'s first Virtual Reality app; a place to discover and enjoy 360 photos and videos on Gear VR.',
+    skills: ['html', 'Ruby on rails', 'css', 'javascript'],
+    skillsModal: ['html', 'css', 'javascript', 'github', 'ruby', 'Bootstraps'],
+    livelink: { link: 'https://felix45.github.io/portfolio', text: 'See live' },
+    seesource: { link: 'https://github.com/felix45/portfolio', text: 'See Source' },
+    seeproject: 'See Project',
+    htmlClass: { article: ['grid-item', 'card', 'flex-item-2', 'hide-ruby'], cardImage: ['card-image'], cardDescription: ['card-description'] },
+    cssStyle: '',
+  },
+
+  {
+    title: 'Uber Navigation',
+    roles: { company: 'Uber', position: ['Lead Developer', '2018'] },
+    image: '/image/Snapshoot-Portfolio(5).png',
```

Comment:
**[REQUIRED CHANGES]**

- [ ] Hey, your project images are not showing as a result of how you've specified the path. Kindly update it to this to make it work..
`'./image/Snapshoot-Portfolio(5).png',
`